crates/table/src/table.rs

```diff
@@ -495,6 +505,45 @@ impl Table {
         Ok((hash, ptr))
     }
 
+    /// Confirms a row update, after first checking constraints.
+    ///
+    /// On `Ok(_)`, statistics of the table are also updated,
+    /// and the `ptr` still points to a valid row, and otherwise not.
```

Comment:
Can you elaborate or explicate "otherwise not?" Is the row deleted in that case? Are the statistics not updated?

Reply:
Doc comment should also mention that this method updates indexes.